doc/doxygen/src/mainpage.md

```diff
@@ -3,6 +3,15 @@ RIOT Documentation                                                  {#mainpage}
 
 [TOC]
 
+Doxygen API Documentation
```

Comment:
```suggestion
RIOT-OS API Documentation
```

Otherwise, one might be tempted to think this is the API doc of Doxygen ;-) (+ end-users rarely care which tool was used to generate the doc)